v2/jdbc-to-spanner/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1,2 @@
+# Required to mock final methods:
+mock-maker-inline
```

Comment:
Do we need this?

Reply:
Will probably be needed in the future so added here